profile/README.md

```
@@ -1,20 +1,29 @@
+| Build   | [Wallets](https://github.com/thirdweb-dev/js) | Embedded wallets, account abstraction, auth, contract interaction, storage, UI components and more |
+| Build   | [Contracts](https://github.com/thirdweb-dev/contracts) | Catalog of prebuilt contracts, Solidity SDK, deployment tools and more |
+| Build   | [Transactions](https://github.com/thirdweb-dev/engine) | Docker container for REST API for transactions, wallet and nonce management, webhooks and more |
+| Monetize  | [Payments](https://github.com/thirdweb-dev/js/tree/main/packages/thirdweb/src/bridge) | Complete payment solution with integrated fiat on-ramp,  |
+| Monetize  | Tokens | Deploy ERC-20, ERC-721, and ERC-1155 tokens with simple configuration, distribution, and listing to marketplaces |
+| Scale  | [Insight](https://github.com/thirdweb-dev/insight) | Real-time on-chain data through REST APIs with flexibility of subgraphs  |
+| Scale  | Account Abstraction | Gas sponsorship through smart wallets with EIP-4337 or EIP-7702 |
+| Scale  | RPC | High-performance access to blockchain networks through globally distributed edge servers |
+| Scale  | Vault | Non-custodial, secure key management service for securing server or developer wallets |
+
+## APIs
+| Platform     | Documentation                                      |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)